README.md

```diff
@@ -21,23 +21,13 @@ THRESHOLD_MEAN_MILLIS - the maximum allowed average response time, in ms
 Note that in order to prevent cloudfoundry from auto-healing and restarting the application,
 the jar will not stop once the tests are complete, but will wait to be stopped externally (this is managed by scripts in htbhf-continuous-delivery).
 
+See the [`run_performance_tests`](https://github.com/DepartmentOfHealth-htbhf/htbhf-continous-delivery/blob/master/cd_scripts/cd_functions.sh#L171)
+function in htbhf-continuous-delivery to see how the performance tests are deployed and run in the CD pipeline.
+
 Note also that the session details app must be running for performance tests to succeed (this provides access to the 2fa confirmation code).
 See [src/test/session-details-provider/README.md](https://github.com/DepartmentOfHealth-htbhf/htbhf-applicant-web-ui/blob/master/src/test/session-details-provider/README.md)
 in htbhf-applicant-web-ui for more details.
 
-## Running the tests from the command line
-First make sure the necessary environment variables are set:
- * BIN_DIR (directory to install gatling to)
- * GATLING_FOLDER_NAME (the name of the folder gatling unzips to, e.g. gatling-charts-highcharts-bundle-3.0.1.1)
- * GATLING_URL
- * UI_APP_NAME (the name of the UI application as shown in cloud foundry, e.g. help-to-buy-healthy-food-staging)
- * CF_PUBLIC_DOMAIN
- * PERF_TESTS_DIR the directory where these scripts reside (e.g. the current directory)
- * RESULTS_DIRECTORY - the directory (relative path) to store the results to.
- * BASE_URL - the url of the UI application to run the tests against.
- * SESSION_DETAILS_BASE_URL - the url of the session details app (this will need to be started before you run the performance tests)
- * PERF_TEST_START_NUMBER_OF_USERS - the number of new users per second at the start of the test
- * PERF_TEST_END_NUMBER_OF_USERS - the number of new users per second at the end of the test
- * PERF_TEST_SOAK_TEST_DURATION_MINUTES - (default 2) - the number of minutes to maintain that level of load, after ramping up
- 
+## Running the tests against your local machine
+First start the session-details-app locally (see above).
 Then run `./run_performance_tests.sh`
```

Comment:
Surely this should be `./run_performance_test_locally.sh`.?

Reply:
Changed, thanks.